pkg/dispatchcli/cmd/update_api.go

```diff
@@ -0,0 +1,162 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/api-manager/gen/client/endpoint"
+	"github.com/vmware/dispatch/pkg/api-manager/gen/models"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+)
+
+var (
+	updateAPILong    = "update an api based on json"
+	updateAPIExample = `dispatch update api my_api --path /new/path`
+
+	httpsOnlyStr string
```

Comment:
The ones defined in create_api.go are booleans.  A user can't do something like `dispatch update api myapi --https-only false` Cobra apparently doesn't recognize values for these boolean flags. If the flag is used then the boolean is set to true.

Reply:
OK... before going further on more endpoints, let's talk with the team about how this should work.  I don't think we were really thinking everything through initially.  The problem I want to avoid is now the CLI is a bit inconsistent in that on create they are flags and on update they take values.  There are other ways of addressing this, and we should explore.  For instance, kubectl has `edit` which just brings you directly into an editor of the resource's document.